src/Tasks/Microsoft.NET.Build.Tasks/GetDependsOnNETStandard.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Build.Framework;
+using System;
+using System.IO;
+using System.Security;
+
+namespace Microsoft.NET.Build.Tasks
+{
+    /// <summary>
+    /// Determines the assembly version to use for a given semantic version.
+    /// </summary>
+    public partial class GetDependsOnNETStandard : TaskBase
```

Comment:
@dsplaisted and I discussed and decided to move it.  I'll need to create a separate test project to do that.